conf/flight_plans/tudelft/rotating_wing_EHVB.xml

```diff
@@ -68,16 +68,29 @@
       <attitude pitch="0" roll="0" throttle="0" until="FALSE" vmode="throttle"/>
     </block>
     <block name="Start Engine" strip_button="Start Engines" strip_icon="on.png">
+        <exception cond="stage_time > 10" deroute="Holding point"/>
         <call_once fun="rotwing_request_configuration(ROTWING_CONFIGURATION_HOVER)"/>
+        <call_once fun="nav_set_heading_current()"/>
         <call_once fun="NavResurrect()"/>
-        <attitude pitch="0" roll="0" throttle="0" until="stage_time>10" vmode="throttle"/>
+        <!-- <set var="nav.horizontal_mode" value="NAV_HORIZONTAL_MODE_ATTITUDE"/>
+        <set var="stabilization_cmd[0]" value="0"/>
+        <set var="stabilization_cmd[1]" value="0"/>
+        <set var="stabilization_cmd[2]" value="0"/>
+        <set var="stabilization_cmd[3]" value="0"/>
+        <while cond="block_time @LT 10"/> -->
+        <attitude pitch="DegOfRad(stateGetNedToBodyEulers_f()->theta)" roll="DegOfRad(stateGetNedToBodyEulers_f()->phi)" throttle="0" until="rotwing_state_hover_motors_running()" vmode="throttle"/>
```

Comment:
I think it would be better to store it and then maintain it than to update it every step.

```suggestion
        <set var="nav.roll" value="DegOfRad(stateGetNedToBodyEulers_f()->phi)"/>
        <set var="nav.pitch" value="DegOfRad(stateGetNedToBodyEulers_f()->theta)"/>
        <attitude pitch="nav.pitch" roll="nav.roll" throttle="0" until="rotwing_state_hover_motors_running()" vmode="throttle"/>
```

Reply:
Re-implements: #3269

Reply:
No this was actually the idea. If you're on the ship, the roll and pitch angle of the drone may continuously change. When running in idle it shouldn't counter steer that.
Do you foresee a problem with this implementation?